### ISSUE TITLE
[BUG] Press F1 to get help page should recongize symbol `[`

### ISSUE THREAD

**Issue by shrektan** [rstudio/rstudio#6612]
First of all, I use the daily builds RStudio IDE for macOS. It's not the very latest version but is new enough (1.3.879). The issue is well illustrated by the reproducible steps and screenshot below.

### Reproducible Step

- Paste the below code to the IDE editor
- Put the cursor between `abs` and `()` (see below screenshot)
- Press `F1` to get the help page
- Error happens

```r
x <- 1:10
x[base::abs()]
```

### The Error

<img width="425" alt="image" src="https://user-images.githubusercontent.com/8368933/78743060-27e82c00-7990-11ea-889d-81f922bdc7c2.png">

<img width="628" alt="image" src="https://user-images.githubusercontent.com/8368933/78743067-2dde0d00-7990-11ea-8ace-d0a90ec61c91.png">

### Expect

No error. This should be identical to `?base::abs`

**Comment by ronblum** [comment]
@shrektan Thank you for raising this! I tried this with RStudio Desktop 1.3.935 on MacOS 10.15.4. Oddly, it failed the first time, but not the second time. However, I was able reproduce the behavior when trying to use <kbd>F1</kbd> on a package that's not installed. For example, if `dplyr` is not installed, then with
```
dplyr::filter
```
place the cursor at the end and click on <kbd>F1</kbd>—the same error occurs.
